pkg/opensearch/response_parser.go

```diff
@@ -100,40 +100,42 @@ func (rp *responseParser) getTimeSeries(timeField string) (*backend.QueryDataRes
 
 func processRawDataResponse(res *es.SearchResponse, timeField string, queryRes backend.DataResponse) backend.DataResponse {
```

Comment:
Just noticed this - is the "es" in es.SearchResponse a remnant from ElasticSearch code?